src/aks-preview/setup.py

```diff
@@ -9,7 +9,7 @@
 
 from setuptools import setup, find_packages
 
-VERSION = "0.5.149"
+VERSION = "0.5.150"
```

Comment:
150 has been used, if you want to release a new version, please update to 151.

Please also update HISTORY.rst and leave some notes.